docs/end-to-end-local-email-dev.md

```diff
@@ -182,48 +207,44 @@ The steps to setup S3 transfer:
 By adding the encryption key first, the AWS console will be able to add
 permissions as we use it.
 
-* Load the [Customer managed keys][customer-managed-keys] page, and select "Create Key"
-    * Step 1: Configure key
-        * Key type: Symmetric
-        * Advanced options: defaults are OK:
-            - Key material origin: KMS
-            - Regionality: Single-Region key
-        * Click "Next"
-    * Step 2: Add labels
-        * Alias: RelayKey or similar
-        * Description: This key is used to encrypt incoming SES messages processed by SNS, SQS, and S3.
-        * Tags: *None*
-        * Click "Next"
-    * Step 3: Define key administrative permissions
-        * Key administrators: Add your login user, if applicable
-        * Key deletion: Select Allow key administrators to delete this key (default)
-        * Click "Next"
-    * Step 4: Define key usage permissions
-        * This account: Add the app key user, if applicable
-        * Other AWS accounts: *None*
-    * Step 5: Review
-        * Add the statement below to the key policy
-        * Click "Finish"
-
-This Key Policy statement (change ``111122223333`` to your account number)
+- Load the [Customer managed keys][customer-managed-keys] page, and select "Create Key"
+  - Step 1: Configure key
+    - Key type: Symmetric
+    - Advanced options: defaults are OK:
+      - Key material origin: KMS
+      - Regionality: Single-Region key
+    - Click "Next"
+  - Step 2: Add labels
+    - Alias: RelayKey or similar
+    - Description: This key is used to encrypt incoming SES messages processed by SNS, SQS, and S3.
+    - Tags: _None_
+    - Click "Next"
+  - Step 3: Define key administrative permissions
+    - Key administrators: Add your login user, if applicable
+    - Key deletion: Select Allow key administrators to delete this key (default)
+    - Click "Next"
+  - Step 4: Define key usage permissions
+    - This account: Add the app key user, if applicable
+    - Other AWS accounts: _None_
+  - Step 5: Review
+    - Add the statement below to the key policy
+    - Click "Finish"
+
+This Key Policy statement (change `111122223333` to your account number)
 allows SES to access the key. Add it to key policy with the other statements:
 
 ```json
 {
   "Sid": "AllowSESToEncryptMessagesBelongingToThisAccount",
   "Effect": "Allow",
   "Principal": {
-    "Service":"ses.amazonaws.com"
+    "Service": "ses.amazonaws.com"
```

Comment:
suggestion (non-blocking): Could reduce code churn here too by NOT re-formatting the JSON inside the json blocks.

Reply:
Yep- there's a precommit hook enforcing it. I'll disable it for this (we can merge, etc) and file a ticket for updating all of the docs as suggested + add that to my sprint.